cabal.project

```diff
@@ -0,0 +1,8 @@
+-- Keep the index-state in tune with bot-plutus-interface
+index-state: 2022-01-22T00:00:00Z
+
+packages: ./*.cabal
+
+--tests: true
```

Comment:
why the --?

Reply:
That was temporary. During the initial nix conversation I needed to make a `cabal.project` file because... I don't remember, but I'm sure I had to. I wanted to leave that commented as a reminder to myself to, uh, hook up the tests.